packages/bot-web-ui/src/components/dashboard/dashboard-component/load-bot-preview/index.scss

```diff
@@ -479,3 +558,76 @@
         width: calc(100vw - 4.8rem) !important;
     }
 }
+
+.dc-dialog {
+    @include mobile {
+        &__wrapper--preview {
+            top: 6.5rem;
+
+            .dc-dialog {
+                &__dialog {
+                    justify-content: flex-start;
+                    min-width: 100vw;
+                    min-height: 100vh;
+                    padding: 0;
+                }
+
+                &__header {
+                    &-wrapper {
+                        margin-bottom: 3.2rem;
+
+                        h1 {
+                            width: 100%;
+                            text-align: center;
+                        }
+
+                        display: flex;
+                        justify-content: center;
+                        align-items: center;
+                        height: 4rem;
+                        box-shadow: inset 0px -1px 0px #f2f3f4;
+                    }
+
+                    &--close {
+                        padding: 1.6rem;
+                        margin-top: -1rem;
+                    }
+                }
+
+                &__content {
+                    max-width: unset;
+
+                    .injectionDiv {
+                        margin: 2.4rem;
+                        height: calc(100vh - 24.5rem);
+                        border: 1px solid #d6d6d6;
+                    }
+
+                    .load-strategy {
+                        &__preview-workspace {
+                            &-container {
+                                position: relative;
+                                margin-bottom: 3.2rem;
+                                min-height: 33rem;
+                            }
+
+                            &-controls {
+                                right: 3.6rem;
+                            }
+                        }
+
+                        &__button-group {
+                            box-shadow: inset 0px 2px 0px #f2f3f4;
```

Comment:
can we use color variables here in this css @rupato-deriv

Reply:
done ✅